src/status_im2/contexts/wallet/send/select_address/view.cljs

```diff
@@ -130,19 +100,20 @@
          :keyboard-should-persist-taps :handled
          :render-fn                    suggestion-component}]])))
 
-(defn- f-view-internal
+(defn- f-view
   []
   (let [margin-top     (safe-area/get-top)
-        selected-tab   (reagent/atom (:id (first tabs-data)))
+
         on-close       (fn []
                          (rf/dispatch [:wallet/clean-scanned-address])
                          (rf/dispatch [:wallet/clean-local-suggestions])
                          (rf/dispatch [:navigate-back]))
-        on-change-tab  #(reset! selected-tab %)
+        on-change-tab  #(rf/dispatch [:wallet/select-address-tab %])
         input-value    (reagent/atom "")
         input-focused? (reagent/atom false)]
     (fn []
-      (let [valid-ens-or-address? (boolean (rf/sub [:wallet/valid-ens-or-address?]))]
+      (let [selected-tab          (or (rf/sub [:wallet/send-tab]) (:id (first tabs/data)))
```

Comment:
changed to using rf here as we need to persist the selected tab for navigating back etc

Reply:
Nice, just make sure that when we navigate back and forth from this screen, the re-frame entry is reset, so the focused tab is the first one.

Reply:
is that what we want? I would have thought we want to be on the same tab we left from? 🤔

Reply:
:thinking: when you go back I think it's expected to keep the state, but when you go forward I think the user expects to start from a "new" state.

But it might be just a personal perspective :thinking: